src/platform/forms-system/src/js/web-component-patterns/checkboxGroupPattern.jsx

```diff
@@ -145,7 +145,6 @@ export const checkboxGroupUI = ({
       }
       return {
         data,
-        label: title,
```

Comment:
dev note: this took `title` at face value and didn't account for updateSchema or updateUiSchema. as you can see in the first file in this PR, we will only accept `data` from `ui:confirmationField` now and not `label`, because label is already correct determined by `ChapterSectionCollection`